### DISCUSSION TITLE
Open GitHub Sponsorship for Mantine?

### DISCUSSION THREAD

**Discussion by reorx** [mantinedev/mantine#1061] [Kind words]
Hi Mantine devs, I've been using Mantine 4.0 for 2 weeks, and it definitely becomes my favorite components library, I want to make some donations to say thanks to you, and I think many of your users will do the same. Please open GitHub Sponsorship for Mantine, so that we can help make this project sustainable and go further.

**Comment by rtivital** [comment]
GitHub's sponsorship program as well as most other services are not available in the country I live in.

**Reply by n3ih7** [reply]
Hi @rtivital, is there any updates on the donation options? Really appreciate everything here.

**Reply by rtivital** [reply]
No, I do not have any updates about this

**Comment by reorx** [comment]
Sorry to hear that, I hope everything well is going well for you!
